app/controllers/admin/omniauth_callbacks_controller.rb

```diff
@@ -3,22 +3,6 @@
 module Admin
   class OmniauthCallbacksController < Admin::ApplicationController
     def setup
```

Comment:
if we are leaving this here becase we might want it later we should leave a comment to indicate this. (like im sure we will need omniauth at some point if we dont already)